### ISSUE TITLE
Vulnerability issue: Prototype Pollution in micromatch@3.1.10

### ISSUE THREAD

**Issue by VlkaFenryka** [gulpjs/findup-sync#48]
**What operating system, Node.js, and npm version?**
OS: Win 10 x64 build 1809
Node: 10.16.0
NPM: 6.9.0
findup-sync 3.0.0

**The Problem**
https://www.sourceclear.com/vulnerability-database/security/prototype-pollution/javascript/sid-20566
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-10746

**Solution**
update to micromatch >= 4.0.0
https://github.com/micromatch/micromatch/blob/master/CHANGELOG.md#400---2019-03-20
https://github.com/micromatch/micromatch/compare/3.1.10...4.0.2

**Comment by phated** [comment]
This is an npm problem. They aren't properly reporting that the issue has been fixed by a path within semver range. See https://github.com/micromatch/micromatch/issues/174

That being said, I'll be releasing a major bump of this with the micromatch major bump once I have some free time.

**Comment by kibertoad** [comment]
@phated 
There is another one related to micromatch:
```
    └─┬ findup-sync@3.0.0
      └─┬ micromatch@3.1.10
        └─┬ snapdragon@0.8.2
          └─┬ base@0.11.2
            └─┬ cache-base@1.0.1
              ├── set-value@2.0.1 
              └─┬ union-value@1.0.1
                └── set-value@2.0.1  deduped
```

Was it also resolved?

**Comment by phated** [comment]
This was just published as 4.0.0

Please note that this is a breaking change due to dropping old node support AND micromatch might have different, unexpected behavior due to its major bump.
